package.json

```diff
@@ -55,7 +55,7 @@
     "mocha": "^6.2.0",
     "nyc": "^13.1.0",
     "ts-node": "^8.3.0",
-    "typescript": "^3.5.3"
+    "typescript": "^3.7.4"
```

Comment:
we should keep evannetwork/dbcp and evannetwork/api-blockchain-core typescript versions aligned...